client/src/components/AvatarDisplay/AvatarDisplay.tsx

```diff
@@ -4,10 +4,21 @@ import { User } from '../../interface/User';
 interface Props {
   loggedIn: boolean;
   user: User;
+  photoUrl?: string;
+  width?: number;
+  height?: number;
 }
 
-const AvatarDisplay = ({ user }: Props): JSX.Element => {
-  return <Avatar alt="Profile Image" src={`https://robohash.org/${user.email}.png`} />;
+const AvatarDisplay = ({ user, width, height, photoUrl }: Props): JSX.Element => {
+  if (photoUrl !== '') {
+    return <Avatar sx={{ width, height }} alt="Profile Image" src={photoUrl} style={{ top: '180px' }} />;
+  } else {
+    return (
+      <Avatar sx={{ width, height }} alt="Profile Image " style={{ top: '180px' }}>
+        {user.name[0]}
+      </Avatar>
```

Comment:
From a design perspective, would a generic avatar image be better like the pet sitter paw print?